service_identity/src/main/kotlin/org/thechance/service_identity/domain/usecases/util/error_codes.kt

```diff
@@ -1,4 +1,4 @@
-package org.thechance.service_identity.endpoints.validation
+package org.thechance.service_identity.domain.usecases.util
 
```

Comment:
remove unnecessary codes and make it in order

Reply:
Done